src/App.vue

```diff
@@ -1,12 +1,24 @@
 <template>
   <div id="app">
-    <h1>BridgeIT</h1>
+    <Navi/>
```

Comment:
Are you sure that `Navi` is proper name for tag?

Navi means US Marines

Reply:
When i try nav be a conflict with html5 tag

Reply:
What about Navigation or something?